feature-libs/product/image-zoom/components/product-image-zoom/product-image-zoom-view/product-image-zoom-view.component.html

```diff
@@ -1,8 +1,11 @@
 <ng-container *ngIf="mainImage$ | async as main">
   <div class="cx-main-image-group" *ngIf="thumbnails$ | async as thumbs">
-    <div class="cx-navigate-image">
+    <div class="cx-navigate-image" aria-labelledby="previouButton">
```

Comment:
This will likely result in the label being repeated twice. Is the aria-label on the button itself not enough?